sema/commons/rdf_ogm/object_graph_mapper.py

```
@@ -0,0 +1,148 @@
+        """
+        if not (a or label or property) and not typed:
+            return None  # can't give you anything if you give me nothing
+
+        if not identifier:
+            if kind == "iri":
+                identifier = f":{uuid4()}"
+            else:
+                identifier = str(uuid4())
+
+        if kind == "iri":
+            identifier = IRITemplate(identifier).expand(self)
```

Ruff (0.12.2)

Undefined name `IRITemplate`

(F821)